lib/membrane_rtc_engine/endpoints/webrtc/connection_allocator.ex

```diff
@@ -20,10 +26,18 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.ConnectionAllocator do
   @doc """
   Function called by the TrackReceiver to register itself in the allocator
   """
-  @callback register_track_receiver(pid(), pid()) :: :ok
+  @callback say_hello(pid(), number(), Track.t()) :: :ok
```

Comment:
That's a nitpick but I would return to the previous version. I think that in the future we can have more processes that will require registration from other processes